content/news/known-amendments.md

```diff
@@ -214,6 +215,18 @@ Fixes a bug that resulted in transaction processing not using new `STAmountCalcS
 With this amendment, the new `STAmountCalcSwitchovers` code applies, which may cause slight changes to transaction processing due to calculation differences. Amounts may be rounded differently and offers may be executed in a different order as a result.
 
 
+## fix1515
+[fix1515]: #fix1515
+
+| Amendment ID                                                     | Status    |
+|:-----------------------------------------------------------------|:----------|
+| 5D08145F0A4983F23AFFFF514E83FAD355C5ABFBB6CAB76FB5BC8519FF5F33BE | Planned   |
+
+Fixes a bug that can occur when a cross-currency payment attempts to consume a large number of offers at the same exchange rate. (Also applies to offers if [FlowCross][] is enabled.) Without the amendment, payment processing gives up on using particular order books if processing the order book would require consuming over 2000 offers.
+
+With this amendment, if a transaction processes over 1000 offers at the same exchange rate, the payment correctly consumes the liquidity from those offers, then moves on to other paths or exchange rates to finish processing the transaction.
+
```

Comment:
I wouldn't characterize the current behavior as a bug. It's fine behavior as long as offer crossing would clear the offers. When FlowCross activates, we want to make sure the offers would still clear. Instead of having special behavior in payments that checked if it was in "offer crossing mode" or not, we decided to change the behavior of payments so it matched offer crossing more closely.

So I'd rephrase this to emphasize that we're aligning the behavior of offer crossing and payments. Maybe something like:
```
Removes a difference in how payment processing and offer crossing consume liquidity. Without this fix, payment processing gives up on using particular order books if processing the order book would require consuming over 2000 offers at the same exchange rate. It would not use the liquidity from those offer and the path containing those offers would not be considered for new liquidity.

With this amendment, if a transaction processes would require consuming over 1000 offers at the same exchange rate, the payment will consume the liquidity from the first 1000 of those offers. After the liquidity is consumed the path containing those offers would not be considered for new liquidity. 
```